#! /bin/sh /usr/share/dpatch/dpatch-run
## 125-fix-PLINAME-creation.dpatch by Kevin B. McCarty <kmccarty@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix creation of PLINAME Make variable so that -DDEBIAN_VERSION=...
## DP: will not get incorporated into it (confusing patchy).

@DPATCH@
diff -urNad cernlib-2006.dfsg.2~/src/patchy/Imakefile cernlib-2006.dfsg.2/src/patchy/Imakefile
--- cernlib-2006.dfsg.2~/src/patchy/Imakefile	2006-09-06 07:36:54.000000000 -0700
+++ cernlib-2006.dfsg.2/src/patchy/Imakefile	2008-02-26 13:13:47.000000000 -0800
@@ -49,7 +49,7 @@
 comma:= ,
 empty:= 
 space:= $(empty) $(empty)
-PLINAME=$(subst $(space),$(comma),$(strip $(subst -DCERNLIB_,,CernlibSystem)))
+PLINAME=$(subst $(space),$(comma),$(strip $(subst -DCERNLIB_,,$(filter -DCERNLIB_%,CernlibSystem))))
 LDIR=$(TOP)$(CURRENT_DIR)/
 
 $(P5LIB): $(LDIR)patchy.car $(LDIR)p5lib.cra p5lib.libmake fcasplit Makefile
@@ -114,7 +114,7 @@
 
 #ifdef CERNLIB_VAXVMS
 
-PLINAME=$(subst $(space),$(comma),$(strip $(subst -DCERNLIB_,,CernlibSystem)))
+PLINAME=$(subst $(space),$(comma),$(strip $(subst -DCERNLIB_,,$(filter -DCERNLIB_%,CernlibSystem))))
 LDIR=$(TOP)$(CURRENT_DIR)]
 
 %.for: %.cra patchy.car yyexpand.com
